SourceX/DiabloUI/diabloui.cpp

```diff
@@ -489,6 +482,18 @@ void LoadUiGFX()
 	LoadHeros();
 }
 
+void UnloadUiGFX()
+{
+	ArtHero.Unload();

```

Comment:
Is there any good reason ArtHero exists outside of selhero.cpp?

Reply:
Probably not, but that's for a separate cleanup